src/Components/AddCar/AddCar.jsx

```diff
@@ -0,0 +1,144 @@
+/* eslint-disable camelcase */
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
+import axios from 'axios';
+import styles from './AddCar.module.css';
+
+const AddCar = () => {
```

Comment:
- [ ] Testing your app, I was able to do this:

![image](https://user-images.githubusercontent.com/57421823/194128126-eef32ef1-a6e5-42ff-a57f-a5b0368e8397.png)

If a user is able to create multiple copies of the same car, with the same name, this could lead to confusion, unintended app functionality, or even bugs in the present or future, even if they have different ids.

I strongly recommend that you provide some validation so a user is not permitted to create 2 cars with the same exact name.

You could do this at Rails level with model validation, or simply with Js on the frontend (insecure though), or both.

That way, your app will behave always as intended 💪

Reply:
We considered making this feature but we thought one user can have 2 cars of the same model and it shouldn't be a problem. Thank you for the suggestion 💯